Gitbook error: "TypeError: cb.apply is not a function"

In addition to #31 there's also another error when using ``docker-compose run --rm build``

![image](https://user-images.githubusercontent.com/8036727/103296308-27983100-49f6-11eb-9aa8-f97f14050137.png)

cc @PauloASilva

```
 ---> e76d1c1c2ca4
Step 4/7 : RUN cd /usr/local/lib/node_modules/gitbook-cli/node_modules/npm/node_modules/ &&     npm install graceful-fs@latest --save
 ---> Running in eac4f3c88243
npm ERR! code EISDIR
npm ERR! syscall copyfile
npm ERR! path /usr/local/lib/node_modules/gitbook-cli/node_modules/npm/node_modules/glob
npm ERR! dest /usr/local/lib/node_modules/gitbook-cli/node_modules/npm/node_modules/.glob-OKa8tj7S
npm ERR! errno -21
npm ERR! EISDIR: illegal operation on a directory, copyfile '/usr/local/lib/node_modules/gitbook-cli/node_modules/npm/node_modules/glob' -> '/usr/local/lib/node_modules/gitbook-cli/node_modules/npm/node_modules/.glob-OKa8tj7S'

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-12-29T16_55_21_408Z-debug.log
ERROR: Service 'build' failed to build: The command '/bin/sh -c cd /usr/local/lib/node_modules/gitbook-cli/node_modules/npm/node_modules/ &&     npm install graceful-fs@latest --save' returned a non-zero code: 235
prompt> docker exec -ti e76d1c1c2ca4 /bin/sh
# # cat /root/.npm/_logs/2020-12-29T16_55_21_408Z-debug.log
cat: /root/.npm/_logs/2020-12-29T16_55_21_408Z-debug.log: No such file or directory
# cd /usr/local/lib/node_modules/gitbook-cli/node_modules/npm/node_modules/glob
# ls -la
total 80
drwxr-xr-x  3 root staff  4096 Dec 29 16:54 .
drwxr-xr-x 99 root staff  4096 Dec 29 16:54 ..
-rw-r--r--  1 root staff   765 Jul  5  2017 LICENSE
-rw-r--r--  1 root staff 14664 Jul  5  2017 README.md
-rw-r--r--  1 root staff  1430 Jul  5  2017 changelog.md
-rw-r--r--  1 root staff  6167 Jul  5  2017 common.js
-rw-r--r--  1 root staff 19433 Jul  5  2017 glob.js
drwxr-xr-x  5 root staff  4096 Dec 29 16:54 node_modules
-rw-r--r--  1 root staff  2124 Jul  6  2017 package.json
-rw-r--r--  1 root staff 12002 Jul  5  2017 sync.js
# id
uid=0(root) gid=0(root) groups=0(root)
#
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM node:current-slim
+FROM node:12.11.1-slim
 
 RUN apt-get update && \
     apt-get install -y git calibre
@@ -15,4 +15,3 @@ RUN git clone -b fix/filename-regex https://github.com/PauloASilva/gitbook.git ~
     gitbook alias ~/gitbook-custom latest
 
 WORKDIR /build
-

